### ISSUE TITLE
'ImageItemList' object has no attribute 'no_split'

### ISSUE THREAD

**Issue by garimss** [fpingham/dataset-cleaner#3]
# We create a databunch with all the data in the training set and no validation set (DatasetFormatter uses only the training set)
db = (ImageItemList.from_folder(path)
                   .no_split()
                   .label_from_folder()
                   .transform([crop_pad, crop_pad], size=224)
                   .databunch())
this gives an error. Anyone have solution for this

**Comment by nazmiasri95** [comment]
I updated to latest fastai v1 and replace it with:

```
tfms = [crop_pad(), crop_pad()]
db = (ImageList.from_folder(path)
                   .no_split()
                   .label_from_folder()
                   .transform(tfms=tfms, size=224)
                   .databunch())
```
